docs/docs/process-payments-with-stripe.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Process Payments with Stripe
+---
+
+## Why Stripe and Gatsby?
+
+Gatsby is an excellent choice for an e-commerce website because static sites are secure, incredibly fast, and easy to host. Whether you're running an online store, accepting donations for a charity, or billing based on usage, you'll need a way to accept payments. Payment processing platforms like Paypal and Stripe are ideal solutions for these use cases.
```

Comment:
We want to [avoid claiming that anything is "easy"](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#avoid-easy-and-simple). Could you be more specific here?

Reply:
Yeah, I'll update that. I took this from the [gatsby e-commerce tutorial](https://www.gatsbyjs.org/tutorial/ecommerce-tutorial/), so that should probably be changed too.

Reply:
I just removed it because I don't think it's absolutely necessary and I couldn't think of a succinct way to explain the convenience of hosting a Gatsby site. What are your thoughts?